brax/io/__init__.py

```diff
@@ -12,3 +12,19 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+GFile = None
+try:
+  from tensorflow.io.gfile import GFile
```

Comment:
Good news!  We are removing tensorflow as a dependency here and elsewhere.  So you can omit this entirely.  I know it then raises the question as to why even have this File abstraction - we need it for some internal use cases.

So it's great if File just wraps the python file object and that's it.

Reply:
done